packages/snap/src/service.ts

```diff
@@ -55,9 +64,28 @@ export async function getLxpBalanceForAddress(
     return 0;
   }
   if (chainId === '0xe708') {
-    return getLxpBalanceFromChain(address);
+    return getTokenBalanceFromChain(LXP_CONTRACT_ADDRESS, address);
+  }
+  return getTokenBalanceFromLineascan(LXP_CONTRACT_ADDRESS, address);
+}
+
+/**
+ * Get the LXP-L balance for an address.
+ * @param address - The address to get the LXP-L balance for.
+ * @param chainId - The chain ID.
+ * @returns The LXP balance for the address.
+ */
+export async function getLxpLBalanceForAddress(
+  address: string,
+  chainId: string,
+) {
+  if (!address) {
+    return 0;
+  }
+  if (chainId === '0xe708') {
```

Comment:
I dont see this `chainId` being used at all.

Reply:
Well, this is where we use it: to check the network your MM is connected to.